fetch.bs

```diff
@@ -1256,6 +1256,13 @@ null), run these steps. <var>processBody</var> must be an algorithm accepting a
 
 <hr>
 
+<p>A <dfn export id=concept-body-with-type>body with type</dfn> is a
+<a for=/>tuple</a> that consists of a
+<dfn export for="body with type" id=concept-body-with-type-body>body</dfn> (a <a for=/>body</a>) and
+<dfn export for="body with type" id=concept-body-with-type-type>type</dfn> (a <a for=/>header value</a>).
```

Comment:
Drop the explicit IDs here. For new stuff we want whatever Bikeshed comes up with generally.